src/mscorlib/corefx/System/Globalization/Calendar.cs

```diff
@@ -29,7 +30,7 @@ namespace System.Globalization
     // The calculation of hour/minute/second is moved to Calendar from GregorianCalendar,
     // since most of the calendars (or all?) have the same way of calcuating hour/minute/second.
 
-
+    [Serializable]
```

Comment:
How did you find all the things that need to be annotated? As I assume the source diff would be difficult. 

We will also need to get this change into corert repo.